uk.co.compucorp.civicrm.hrcontactactionsmenu/scss/components/_contact-actions-menu.scss

```diff
@@ -0,0 +1,70 @@
+.crm_contact-actions {
+  padding: 0;
+  width: 1065px !important;
+  height: auto;
+
+  #bootstrap-theme .row {
```

Comment:
this can't be right

1. You are mentioning the spacename
2. You are overriding bootstrap's grid rules